src/bin/cargo/commands/owner.rs

```diff
@@ -6,46 +6,137 @@ use cargo_credential::Secret;
         .arg(
             multi_opt(
                 "add",
                 "LOGIN",
                 "Name of a user or team to invite as an owner",
             )
-            .short('a'),
+            .short('a')
+            .hide(true),
         )
         .arg(
             multi_opt(
                 "remove",
                 "LOGIN",
                 "Name of a user or team to remove as an owner",
             )
-            .short('r'),
+            .short('r')
+            .hide(true),
         )
-        .arg(flag("list", "List owners of a crate").short('l'))
+        .arg(flag("list", "List owners of a crate").short('l').hide(true))
+        .subcommands([
+            for_subcommand_add_arg(
+                Command::new("add")
+                    .about("Name of a user or team to invite as an owner")
+                    .arg_quiet()
+                    .args([
+                        Arg::new("add")
+                            .required(true)
+                            .value_delimiter(',')
+                            .value_name("OWNER_NAME")
+                            .help("Name of the owner you want to invite"),
+                        Arg::new("crate")
+                            .value_name("CRATE_NAME")
+                            .help("Crate name that you want to manage the owner"),
+                    ]),
+            ),
+            for_subcommand_add_arg(
+                Command::new("remove")
+                    .about("Name of a user or team to remove as an owner")
+                    .arg_quiet()
+                    .args([
+                        Arg::new("remove")
+                            .required(true)
+                            .value_delimiter(',')
+                            .value_name("OWNER_NAME")
+                            .help("Name of the owner you want to remove"),
+                        Arg::new("crate")
+                            .value_name("CRATE_NAME")
+                            .help("Crate name that you want to manage the owner"),
+                    ]),
+            ),
+            for_subcommand_add_arg(
+                Command::new("list")
+                    .about("List owners of a crate")
+                    .arg_quiet()
+                    .arg(
+                        Arg::new("crate")
+                            .value_name("CRATE_NAME")
+                            .help("Crate name which you want to list all owner names"),
+                    ),
+            ),
+        ])
         .arg_index("Registry index URL to modify owners for")
         .arg_registry("Registry to modify owners for")
         .arg(opt("token", "API token to use when authenticating").value_name("TOKEN"))
-        .arg_quiet()
         .after_help(color_print::cstr!(
             "Run `<cyan,bold>cargo help owner</>` for more detailed information.\n"
         ))
 }
 
+fn for_subcommand_add_arg(command: Command) -> Command {
```

Comment:
This is a confusing function name.  My first thought was "why are `add` subcommand args being applied to `remove` and `list`?

Maybe `add_registry_args`?

Reply:
Sorry, it has been modified.